app/views/styles/care-cards.html

```diff
@@ -9,12 +10,15 @@ <h1>Care Cards</h1>
 {% endblock %}
 
 {% block sidebar %}
-    <h2 class="nhsuk-heading-xs">Contents</h2>
-    <ul class="">
-		<li><a href="#care_card_111">111 Care Card</a></li>
-		<li><a href="#care_card_primary">Primary Care Card</a></li>
-		<li><a href="#care_card_urgent">Urgent Care Card</a></li>
-		<li><a href="#care_card_emergency">Emergency Care Card</a></li>
+    <h2 class="nhsuk-heading-xs">Page contents</h2>
+    <ul>
```

Comment:
Don't know what to do about these internal links as they don't go anywhere? Not a showstopper.

Reply:
I've just added those as placeholders I'm hoping David or Ben will update the content of these pages following this.